### ISSUE TITLE
Accessibility errors found in your template

### ISSUE THREAD

**Issue by carlsonsantana** [wapyce/test#4]
The user @admin validate your site template "https://nandomoreirame.github.io/mug/" and found these accessibility errors:
* Page https://nandomoreirame.github.io/mug/:
  * Issue *3512aa3d-9474-431a-8ba1-2c12c77c856f*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.A.NoContent
    * **Message**: Anchor element found with a valid href attribute, but no link content has been supplied.
    * **Selector**: `html > body > a`
    * **Context**: `<a href="http://github.com/nandomoreirame/mug" target="_blank" class="github-corner"><svg width="80" height="80" vie...</a>`
  * Issue *e38b26b4-e39e-4f9f-8128-5782517d5879*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.18:1. Recommendation: change text colour to #292929.
    * **Selector**: `html > body > main > section > section > nav > ul > li:nth-child(1) > span`
    * **Context**: `<span class="page-number">Page 1 of 4</span>`
  * Issue *c9ab0c24-b85d-4096-8f24-c2535eef3df0*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.18:1. Recommendation: change text colour to #292929.
    * **Selector**: `html > body > main > section > section > article:nth-child(2) > span > time`
    * **Context**: `<time class="post-date" datetime="2015-11-12">Nov 12, 2015 | jekyll update</time>`
  * Issue *8989424e-124f-469f-b059-25960d0bd29f*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_3.1_3_1.F68
    * **Message**: This form field should be labelled in some way. Use the label element (either with a "for" attribute or wrapped around the form field), or "title", "aria-label" or "aria-labelledby" attributes as appropriate.
    * **Selector**: `#search-input`
    * **Context**: `<input type="text" id="search-input" class="input" placeholder="enter the key for search..">`
  * Issue *12653d25-e516-43e2-baa4-929fcb6248e1*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.InputText.Name
    * **Message**: This textinput element does not have a name available to an accessibility API. Valid names are: label element, title undefined, aria-label undefined, aria-labelledby undefined.
    * **Selector**: `#search-input`
    * **Context**: `<input type="text" id="search-input" class="input" placeholder="enter the key for search..">`
  * Issue *d43ca5c6-eddf-4d1c-b8b5-f2f5cb569e98*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.A.NoContent
    * **Message**: Anchor element found with a valid href attribute, but no link content has been supplied.
    * **Selector**: `html > body > main > header > div > a:nth-child(1)`
    * **Context**: `<a href="#" class="nav-toogle js-menu-trigger sliding-panel-button">
      <span></span>
      <spa...</a>`
  * Issue *73d392b4-7f04-4203-b3d3-8b37bd8e365d*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle3.Guideline3_1.3_1_1.H57.2
    * **Message**: The html element should have a lang or xml:lang attribute which describes the language of the document.
    * **Selector**: `html`
    * **Context**: `<html><head>
    <meta charset="utf-8...</html>`

You can check these accessibility errors using [pa11y](https://github.com/pa11y/pa11y).
